### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by aarondr77**
# Changes

- [ ] I ran all the tests to make sure I didn't break anything.

This adds tests for diff, merge, reset personal. And fixes a few minor bugs that were caught along the way.

### CODE REVIEW

**Comment by aarondr77** [comment]
We are not passing two tests currently: merge when adding a column and reset personal deleting what is on the personal version.

File: src/saga/diff.js [hunk 1]
```diff
@@ -46,7 +46,8 @@ function replaceFormulas(formulas, sheetName, commitSheetPrefix) {
             let formula = formulas[i][j];
             if (formula[0] === "=") {
                 // Then, it's a formula, and we try and replace
-                formula = formula.replaceAll("'" + commitSheetPrefix, "'");
+                // TODOD: remove the ' on both sides of the resulting sheet name
+                formula = formula.replaceAll(commitSheetPrefix, "");
```
Comment (naterush) [inline src/saga/diff.js:50]: Much better!

File: src/tests/suites/mergeTests.js [hunk 2]
```diff
@@ -185,6 +93,64 @@ export async function testMergePreservesCrossSheetReferences() {
     return true;
 }
 
+export async function testOriginalEmptyMergeConflict() {
```
Comment (naterush) [inline src/tests/suites/mergeTests.js:96]: What is original? If this is just the name for the old version of the test, try and make it a bit more informative if possible.

**Review by naterush — APPROVED** [review]
This looks great. Very minor changes, feel free to merge in. I think having these breaking these here is fine for now - as a reminder to fix.

**Pull request merged**